subsys/bluetooth/common/Kconfig

```diff
@@ -4,6 +4,187 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config BT_BUF_ACL_TX_SIZE
+	int "Maximum supported ACL size for outgoing data"
+	range 27 251
+	default 27
+	help
+	  Maximum supported ACL size of data packets sent from the Host to the
+	  Controller. This value does not include the HCI ACL header.
+	  The Host will segment the data transmitted to the Controller so that
+	  packets sent to the Controller will contain data up to this size.
+	  In a combined build this value will be set in both the Host and the
+	  Controller.
+	  In a Host-only build the Host will read the maximum ACL size supported
+	  by the Controller and use the smallest value supported by both the
+	  Bost and the Controller.
+	  The Host supports sending of larger L2CAP PDUs than the ACL size and
+	  will fragment L2CAP PDUs into ACL data packets.
+	  The Controller will return this value in the HCI LE Read Buffer
+	  Size command response. If this size if greater than effective Link
+	  Layer transmission size then the Controller will perform
+	  fragmentation before transmitting the packet(s) on air.
+	  If this value is less than the effective Link Layer transmission size
+	  then this will restrict the maximum Link Layer transmission size.
+	  Maximum is set to 251 due to implementation limitations (use of
+	  uint8_t for length field in PDU buffer structure).
+
+config BT_BUF_ACL_TX_COUNT
+	int "Number of outgoing ACL data buffers"
+	default 7 if BT_HCI_RAW
+	default 3
+	range 1 255
+	help
+	  Number of outgoing ACL data buffers sent from the Host to the
+	  Controller. This determines the maximum amount of data packets the
+	  Host can have queued in the Controller before waiting for the
+	  to notify the Host that more packets can be queued with the Number of
+	  Completed Packets event.
+	  The buffers are shared between all of the connections and the Host
+	  determines how to divide the buffers between the connections.
+	  The Controller will return this value in the HCI LE Read Buffer Size
+	  command response.
+
+config BT_BUF_ACL_RX_SIZE
+	int "Maximum supported ACL size for incoming data"
+	default 200 if BT_BREDR
+	# Mesh Proxy Recommended: 64 Pkey + 2 Bytes Mesh header.
+	# Overhead: ATT Write command Header (3) in an L2CAP PDU (4).
+	default 73 if BT_MESH_PROXY
+	default 70 if BT_EATT
+	default 69 if BT_SMP
+	default 27
+	range 70 1300 if BT_EATT
+	range 69 1300 if BT_SMP
+	range 27 1300
+	help
+	  Maximum support ACL size of data packets sent from the Host to the
```

Comment:
should it be "from the Controller to the Host"?

Reply:
Yes, this needs fixing.